vendor/discord-rpc/premake5.lua

```diff
@@ -0,0 +1,23 @@
+project "discord-rpc"
+	targetname "discord-rpc"
+	language "C++"
+	kind "StaticLib"
+
+	includedirs { 
+		"discord/include",
+		"discord/thirdparty/rapidjson/include"
+	}
+
+	files {
+		"premake5.lua",
+		"discord/src/discord_rpc.cpp",
+		"discord/src/rpc_connection.cpp",
+		"discord/src/serialization.cpp",
+		"discord/src/connection_win.cpp",
+		"discord/src/discord_register_win.cpp",
+	}
+
+	filter "architecture:x64"
```

Comment:
Exclude for anything that is not x86 (excludes ARM, ARM64 and x64).
```suggestion
	filter "architecture:not x86"
```